install-from-binstall-release.sh

```diff
@@ -2,6 +2,38 @@
 
 set -euxo pipefail
 
+if [ "${OS-}" = "Windows_NT" ]; then
+    # https://github.com/cargo-bins/cargo-binstall/blob/main/install-from-binstall-release.ps1
+    # shellcheck disable=SC2016
+    powershell -c '$ErrorActionPreference = "Stop"
```

Comment:
Is it possible to fetch the .ps1 script and run it here, instead of embedding it here which makes it harder to maintain?

BTW you can also removemthe "shellcheck disable" line if you adopt this advice.

Reply:
It is. To elaborate on why I didn't do that in the first place: it means that any PRs that are opened or forked copies will _always use the main branch of cargo-binstall_ even if the user changes the `.ps1` script in their custom fork to point to a mirror or something.

This has two sides:
1. There's no copy-pasting each time the `.ps1` script changes
2. It's a gotcha for any forks or PRs that change the `.ps1` script but isn't merged into main

It's up to you which way you want to go.

Reply:
I prefer using the .ps1 since maintaining two copies is going to be painful and it will eventually become out of sync.

I don't think branch we become an issue, we can simply use the `./install-from-script.ps1` instead of downloading it.

Reply:
For the branch issue though, we could override it in CI to run the local one `./install-from-script.ps1`.

For people using a script we can provide an env `BRANCH` and use `BRANCH=${BRANCH:-main}` in our code.